types/src/main/java/org/ethereum/beacon/types/Ether.java

```diff
@@ -27,6 +27,14 @@ public static Ether valueOf(long unsignedValue) {
     return new Ether(UInt64.valueOf(unsignedValue));
   }
 
+  public static Ether ofGWeis(UInt64 gWeis) {
```

Comment:
I think we need too add a separate type for `GWei` later on.

Reply:
Right now doing it

Reply:
👍